beacon_node/store/src/config.rs

```diff
@@ -57,9 +58,16 @@ pub struct StoreConfig {
 }
 
 /// Variant of `StoreConfig` that gets written to disk. Contains immutable configuration params.
-#[derive(Debug, Clone, PartialEq, Eq, Encode, Decode)]
-// FIXME(sproul): schema migration
+#[superstruct(
+    variants(V0, V1),
```

Comment:
Usually the naming convention we use for versioned structs in the database follows the schema version: https://github.com/sigp/lighthouse/tree/stable/beacon_node/beacon_chain/src/schema_change#naming-conventions

```suggestion
    variants(V1, V22),
```

Reply:
Fixed in e5661720685236e73c6fee55998f15ba44a9ed88